src/arith.rs

```diff
@@ -1,9 +1,14 @@
 use core::cmp::Ordering;
-use rand::Rng;
 use crunchy::unroll;
+use rand::Rng;
 
 use byteorder::{BigEndian, ByteOrder};
 
+#[cfg(all(target_os = "zkvm", target_arch = "riscv32"))]
+use bytemuck;
```

Comment:
Is this needed?

Reply:
This crate wants to store field elements as `[u128; 2]`s and our bigint system wants to read/write them as `[u32; 8]`s -- `bytemuck` lets us go between those efficiently.

Reply:
I just mean the `use` statement doesn't seem needed since all cases do `bytemuck::foo`

Reply:
Right, good call